README.md

```diff
@@ -57,7 +57,7 @@ Do you have many different apps in your Android Phone for the purpose of clickin
 
 * Finally, after all this editing you can easily share the image to your favourite social media sites with our easy-to-use sharing feature.
 
-* Facebook, Twitter, NextCloud, OwnCloud, Imgur, Dropbox, Box, Flickr, Pinterest, Instagram, Whatsapp and Tumblr - You name it and we have it covered.
+* Facebook, Twitter, NextCloud, OwnCloud, Imgur, Dropbox, Box, Flickr, Pinterest, Instagram, Whatsapp, and Tumblr - You name it and we have it covered.
```

Comment:
Remove the `,` before `and` that is not necessary.
https://www.grammarly.com/blog/comma-before-and/  this might help

Reply:
Though it is not necessary, I thought it will be perfect if we use "Oxford commas". :)

Reply:
cool, i'll remove it then.